comtypes/__init__.py

```diff
@@ -3,7 +3,7 @@
 import os
 
 # comtypes version numbers follow semver (http://semver.org/)
-__version__ = "1.1.2"
+__version__ = "1.1.3-0.fixIssue92"
```

Comment:
It's not noted here, but I think it is best to follow PEP 440. I recommend `1.1.3.dev0`.